samples/maps/members/axis-update/demo.js

```diff
@@ -34,23 +34,31 @@ Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@v7.0.0/sam
     });
 });
 
-var blackAndWhite = true,
+let blackAndWhite = true,
     log = true;
 
-$('#update-color').click(function () {
-    var colorAxis = chart.colorAxis[0];
+const updateColor = () => {
```

Comment:
Why a named function instead of an anonymous function like before?

Reply:
At the time it made sense, but I see now that it might look cleaner with an anonymous function instead.